chart/apisix/templates/configmap.yaml

```diff
@@ -82,7 +82,9 @@ data:
         - 0.0.0.0/0
       {{- end}}
       #   - "::/64"
-      # port_admin: 9180              # use a separate port
+      {{- if .Values.admin.enabled }}
+      port_admin: {{ .Values.admin.containerPort }}
```

Comment:
Why not use `admin.port`

Reply:
I used to call it admin_port

Reply:
I think `.Values.admin.admin_port` is redundant, maybe `.Values.admin.port` is better.

Reply:
@totemofwolf @lilien1010 Changed.